### DISCUSSION TITLE
在使用-noweb 不启动web服务的情况下如何查看日志

### DISCUSSION THREAD

**Discussion by PKC278** [jeessy2/ddns-go#1051] [Q&A]
如题，我没有找到日志文件，系统标准输入输出中也没有找到相关日志

**Comment by WaterLemons2k** [comment]
有填写配置文件吗？

**Reply by PKC278** [reply]
有的，一切正常，只是想在不启动web的时候看一下日志

**Reply by PKC278** [reply]
没用docker，在openwrt中用的这个插件https://github.com/sirpdboy/luci-app-ddns-go

**Reply by WaterLemons2k** [reply]
> 没用docker，在openwrt中用的这个插件[sirpdboy/luci-app-ddns-go](https://github.com/sirpdboy/luci-app-ddns-go)

不太懂插件，请询问插件提供方

**Reply by PKC278** [reply]
是不是ddns-go不会将日志存到文件，只会输出到标准输出stdout中

**Reply by WaterLemons2k** [reply]
> 是不是ddns-go不会将日志存到文件，只会输出到标准输出stdout中

是的
